andela/workdey
# #[107791640] Tasker log

# [#107791640]

As a tasker
I want to see a log of all tasks that I have created: done, to be done and being done 
because this will help me to know how effectively and easily I'm getting things done on the app.
#### What does this PR do?

Enables a tasker to keep track of tasks done, currently being done and _inactive_
#### Description of Task to be completed?

Given I'm a tasker.
When I click show log button.
Then I should see a page that lists all my tasks and show which ones have been done and which ones are undone.
#### How should this be manually tested?

When you (**tasker**) log in, you click on Tasks logs on your dashboard sidebar and you get to see the log of your tasks
#### What are the relevant pivotal tracker stories?
#107791640
#### Screenshots (if appropriate)

![screen shot 2016-04-28 at 6 05 54 pm](https://cloud.githubusercontent.com/assets/14159416/14979575/2109158e-111b-11e6-9a49-eb78d1422cc2.png)
